profiles/aws/env/settings.yaml

```diff
@@ -6,16 +6,21 @@ alerts:
     channelCrit: mon-otomi-crit
 customer:
   name: demo
-otomi:
-  isMultitenant: true
-  mode: ce
-  teamPrefix: team-
+dns:
+  dnsZones: []
+  domain: demo.eks.otomi.cloud
+  aws:
+    region: eu-central-1
 oidc:
-  adminGroupID: 'te'
+  adminGroupID: te
```

Comment:
can this be `adminGroupID` then? just like `teamAdminGroupID `?

Reply:
sure